src/app/modules/timesheet/services/timesheet.service.spec.ts

```diff
@@ -2,12 +2,29 @@ import { TimesheetService } from './timesheet.service';
 
 import { Timesheet } from 'src/app/shared/models/timesheet.model';
 
+import { TestBed } from '@angular/core/testing';
+
+import { HttpClientTestingModule} from '@angular/common/http/testing';
+import {of} from 'rxjs/internal/observable/of';
```

Comment:
`import { of } from 'rxjs';` is not enough ? (don't forget spaces between braces)